### ISSUE TITLE
Icons not showing in high contrast mode in Contextual menu

### ISSUE THREAD

**Issue by pagangil** [OfficeDev/office-ui-fabric-react#4248]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: Edge, Windows

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: High

Products/sites affected: (if applicable)

#### Describe the issue:

In high contrast mode, if we use contextual menu dropdown with menu items having icons, then the icons are not visible for unselected menu items.

#### Actual behavior:

The icons are not visible for non-focused menu items in contextual menu

#### Expected behavior:

The icons should be visible

### If applicable, please provide a codepen repro:

![image](https://user-images.githubusercontent.com/31500022/37334304-89f67bd6-26d1-11e8-9ef7-2dca56383470.png)

https://developer.microsoft.com/en-us/fabric#/components/contextualmenu

**Comment by betrue-final-final** [comment]
Another good one to investigate @lynamemi

**Comment by lynamemi** [comment]
Note that in the example above, 'Upload' has an inline style applied for color (salmon) so that will always be  salmon color. The PR I have in will fix icons without inline color styles.

**Comment by lynamemi** [comment]
@satbai That is a separate issue. I think what you are showing is Dropdown with focus. You can open another issue, or tag on the screenshot you have to this existing issue:
https://github.com/OfficeDev/office-ui-fabric-react/issues/3488

**Comment by lynamemi** [comment]
fixed in https://github.com/OfficeDev/office-ui-fabric-react/pull/4311
